Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -1758,9 +1785,14 @@ def get_alerts(client: Client, args: dict) -> CommandResults:
         ids=argToList(args.get('alert_ids')),
         limit=int(args.get('limit', 1000)),
         siteIds=args.get('site_ids'),
+        cursor=args.get('cursor'),
     )
 
-    alerts = client.get_alerts_request(query_params)
+    alerts, pagination = client.get_alerts_request(query_params)
+
+    if pagination['nextCursor'] is not None:
```

Comment:
Whats the default page size? Is it configurable?
Please implement paging here according to our standard, which can be found [here](https://xsoar.pan.dev/docs/integrations/code-conventions#pagination-in-integration-commands)